bot/resources/evergreen/py_topics.yaml

```diff
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Was Python your first programming language? If not, what was?
+    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
+    - In what ways has Python Discord helped you with Python?
+    - Are you currently using Python professionally, for education, or as a hobby?
+    - What is your process when you decide to start a project in python?
+    - Have you ever been unable to finish a python project? What was it and why?
+    - How often do you program in python?
+    - How would you approach learning a new Python library?
+    - Have you ever worked with a microcontroller or anything physical with python before?
+    - How good would you say you are at Python so far: beginner, intermediate or advanced?
+    - Have you ever tried making your own programming language?
```

Comment:
- What do you plan on eventually achieving with Python?
    - Was Python your first programming language? If not, what was?
    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
    - In what ways has Python Discord helped you with Python?
    - Are you currently using Python professionally, for education, or as a hobby?
    - What is your process when you decide to start a project in Python?
    - Have you ever been unable to finish a Python project? What was it and why?
    - How often do you program in Python?
    - How would you approach learning a new Python library?
    - Have you ever worked with a microcontroller or anything physical with Python before?
    - What Python level do you think you are at: Beginner, Intermediate or Advanced?
    - Have you ever tried making your own programming language?